.github/workflows/download-translations.yml

```diff
@@ -0,0 +1,137 @@
+name: 'Download translations from Crowdin'
+
+permissions:
+  actions: write
+  checks: write
+  contents: write
+  deployments: write
+  pull-requests: write
+  statuses: write
+  
+  
+on:
+  repository_dispatch:
+    types:
+      - file-fully-reviewed
+
+jobs:
+  synchronize-with-crowdin:
+    runs-on: ubuntu-latest
+    if: github.repository == 'binary-com/deriv-app'
+    steps:
+      - name: Checkout master branch
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+          ref: master
+
+      - name: Get cached dependencies
+        id: cache_npm
+        uses: actions/cache@v3
+        with:
+          key: translations-build-${{ hashFiles('**/package-lock.json') }}
```

Comment:
Caching the dependencies will speed up the workflow everytime the webhook event triggers (from 6.5 minutes to 1.5 minutes):
<img width="1074" alt="Screenshot 2023-07-28 at 12 55 49 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/4c36cbd4-6781-4ea0-b95d-19e680bc8583">